css/style.css

```diff
@@ -0,0 +1,1050 @@
+    align-items: center;
+    background: #f8f8f8;
+  }
+
+  header.about h2 {
+    display: none;
+  }
+
+  header.about h1 {
+    font-size: 2rem;
+    line-height: 2rem;
+    width: 50vw;
+  }
+
+  header.about .header-hr {
+    margin: 0.8rem auto;
+    margin-top: calc(16px + 0.8rem);
+    width: 30px;
+    border: 0.5px solid #ec5242;
+  }
+
+  header.about .about-text {
+    background: #fff;
+    border: 0.5px solid #d3d3d3;
+    width: 80vw;
+    margin-bottom: 2rem;
+  }
+
+  /* About logo styles */
+
+  .about-logo {
+    padding: 2rem 1.4rem;
+  }
+
+  .about-logo div {
+    margin: 0 auto;
+    width: 40vw;
+  }
+
+  .about-logo .the-logo {
+    color: #272a31;
+    font-weight: 400;
+    text-align: center;
+    font-size: 1.2rem;
+  }
+
+  .about-logo .logo-hr {
+    margin: 0.8rem auto;
+    width: 30px;
+    border: 0.5px solid #ec5242;
+  }
+
+  .about-logo .text {
+    font-size: 0.8rem;
+  }
+
+  .about-logo .logo {
+    margin-top: 1rem;
+    padding: 1rem 3rem;
+    border: 1px solid #d3d3d3;
+  }
+
+  .about-logo .logo img {
+    width: 50%;
+    max-width: 100%;
+  }
+
+  /* About planning styles */
+
+  hr.between-sections {
+    border: 0.5px solid #d3d3d3;
+  }
+
+  .about-card {
+    padding: 2rem;
+  }
+
+  .about-card div {
+    width: 70vw;
+    margin: 0 auto;
+  }
+
+  .about-card .possible {
+    font-size: 1.2rem;
+    font-weight: 400;
+  }
+
+  .about-card .planning-hr {
+    margin: 0.8rem auto;
+    width: 30px;
+    border: 0.5px solid #ec5242;
+  }
+
+  .about-card .text {
+    font-size: 0.8rem;
+    line-height: 1rem;
+  }
+
+  .about-card .vids-wrapper {
+    display: flex;
+    justify-content: center;
+    gap: 1rem;
+    width: 100%;
+    padding: 1rem 0;
+    margin: 0;
+  }
+
+  .about-card .vid {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 0;
+    margin-top: 1rem;
+    height: 250px;
+    color: #fff;
+  }
+}
+
+.menu-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  height: 35px;
+  width: 35px;
+  top: 20px;
+  left: 20px;
+  position: sticky;
+  z-index: 5;
+  transition: all 0.5s ease-in-out;
+}
+
+.hamburger {
+  width: 35px;
+  height: 5px;
+  position: absolute;
+  background: #272a31;
+  border-radius: 3px;
+  transition: all 0.5s ease-in-out;
+}
+
+.hamburger::before,
+.hamburger::after {
+  content: '';
+  position: absolute;
+  width: 35px;
+  height: 5px;
+  left: 0;
+  border-radius: 3px;
+  background: #272a31;
+  transition: all 0.5s ease-in-out;
+}
+
+.hamburger::before {
+  transform: translateY(-10px);
+}
+
+.hamburger::after {
+  transform: translateY(10px);
+}
+
+/* Menu button animation */
+.menu-btn.open .hamburger {
+  transform: translateX(-35px);
+  background: transparent;
+}
+
+.menu-btn.open .hamburger::before {
+  transform: rotate(45deg) translate(35px, -35px);
+}
+
+.menu-btn.open .hamburger::after {
+  transform: rotate(-45deg) translate(35px, 35px);
+}
```

Comment:
- [ ] Please always make sure that media queries should be organized at the button of the stylesheet, that will save you from stress of using important! everytime.. Therefore let's move these codes above the media queries.